app/views/root/chromeless.html.erb

```diff
@@ -1,20 +1 @@
-<% content_for :page_title, content_for?(:title) ? yield(:title) : "GOV.UK - The best place to find government services and information" %>
-
-<% content_for :head do %>
-  <%= render :partial => 'stylesheet', :locals => { :css_file => 'header-footer-only' } %>
-<% end %>
-
-<% @omit_header = true %>
-
-<% content_for :content do %>
-  <div id="wrapper" class="govuk-width-container group">
-    <%= yield :wrapper_content %>
-  </div>
-<% end %>
-
-<% content_for :footer_support_links do %>
-  <%= render :partial => 'footer_support_links' %>
-<% end %>
-<% content_for :body_end do %>
-  <%= render :partial => 'javascript', :locals => { :js_file => 'header-footer-only' } %>
-<% end %>
+<%= render partial: "gem_base" %>
```

Comment:
Is showing both the header and footer wanted here? Prior to this change both were left out for this particular template.

Reply:
Super - thanks for explaining. Makes sense; and if there is any special work I agree it'd best be done outside of this pull request.